pkg/controller/appdefinition/volume.go

```diff
@@ -201,27 +207,77 @@ func toPVCs(req router.Request, appInstance *v1.AppInstance) (result []kclient.O
 			}
 		}
 
+		// Ensure that no other PersistentVolume exists with the same public name
+		selector := klabels.SelectorFromSet(map[string]string{
+			labels.AcornPublicName: pvc.Labels[labels.AcornPublicName],
+			// Make sure we only list volumes that are part of the same Acorn project
+			labels.AcornAppNamespace: appInstance.Namespace,
+			labels.AcornManaged:      "true",
```

Comment:
Should this be left out just in case? I think have the namespace makes sense, especially if we use the same selector to list volumes in the API server when GET-ing.

Reply:
I'm a bit confused, are you asking if I should remove the `acorn.io/managed` selector?

Reply:
Yes. My thought is, "What if the user tries to create a volume with the public name and app namespace labels?"

If a user did this, then you wouldn't see that PV here, but it may cause issues if we try to list in other places.

If you think I am being too cautious, then I am fine leaving this as is.